components/Common/Card/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo, type ComponentProps, type FC } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+import { shortHumanReadableDate } from '@/util/shortHumanReadableDate';
+
+import styles from './index.module.css';
+
+type Author = {
+  firstName: string;
+  lastName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type CardProps = {
+  title: ComponentProps<typeof Preview>['title'];
+  type: ComponentProps<typeof Preview>['type'];
+  subtitle: string;
+  description: string;
+  author: Author;
+  date: Date;
+};
+
+const Card: FC<CardProps> = ({
+  title,
+  type,
+  subtitle,
+  description,
+  author,
+  date,
+}) => {
+  const avatars = useMemo(() => {
+    const avatars = [
+      {
+        alt: `${author.firstName} ${author.lastName}`,
+        src: author.src,
+      },
+    ];
+
+    return avatars;
+  }, [author.firstName, author.lastName, author.src]);
```

Comment:
With the solution I mentioned above, there will be no need to make these changes

Reply:
Thanks for your suggestion! As mentioned [here](https://github.com/nodejs/nodejs.org/pull/6037/files#r1367783451), however, I have added these properties for better clarity on what the intended usage of these values is.